src/singletons/Crashpad.hpp

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "common/FlagsEnum.hpp"
+#include "common/Singleton.hpp"
+
+#include <QtGlobal>
+
+#ifdef CHATTERINO_WITH_CRASHPAD
+#    include <client/crashpad_client.h>
+
+#    include <memory>
+#endif
+
+namespace chatterino {
+
+class CrashRecovery : public Singleton
```

Comment:
What is the purpose of the `CrashRecovery` class?

Reply:
It stores the flags for the crash-handler. So it's like `Settings` but for crashes. The settings library can't be used, as the app isn't initialized when loading the flags.